### ISSUE TITLE
Resizing peek window lost code preview.

### ISSUE THREAD

**Issue by Xarkam** [Microsoft/vscode#73019]
On resizing peek code window, the code preview disappears.
Changing to other reference in listing files on the right does not retrieve the code

- VSCode Version: 1.33.1
- OS Version: 
```
Version: 1.33.1
Commit: 51b0b28134d51361cf996d2f0a1c698247aeabd8
Date: 2019-04-11T08:20:22.771Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.18.0-18-generic snap
```

Steps to Reproduce:

1. Click on code reference to show peek window
2. Resize peek preview code window

Does this issue occur when all extensions are disabled?: Yes

**Comment by jrieken** [comment]
I cannot reproduce. Is there anything in the error console (F1 > Dev Tools > Console) or any other error showing? Do you resize the height or the middle sash? Can you try to reproduce with latests insider?

**Comment by Xarkam** [comment]
Here is what I did with the Insider version.
I installed one by one all the extensions I use in the stable version and at each installation I checked the behavior without reproducing the bug.
I then erased all the extensions to copy the extensions of the stable version to insider without reproducing the bug.
I erased all the configuration files from the insider version and copied the files from the stable version without reproducing the bug.

The only mistake I have in the stable and insider version is this:
```
gitExec:err:Failed to execute git {
  "exitCode": 1,
  "gitCommand": "config",
  "stdout": "",
  "stderr": ""
  (at GitProvider.gitExec (/home/xarkam/.vscode-insiders/extensions/ms-vsliveshare.vsliveshare-1.0.91/src/contactService/providers/GitProvider.ts:175:21))
```
I analyzed all the logs files without revealing any error except for the error mentioned above.

When I open the developers tools, I have this error (stable version and insider):
Can not find an application or module for chrome-devtools://devtools/bundled/toolbox.html?remoteBase=https://chrome-devtools-frontend.appspot.com/serve_file/@164c37e3f235134c88e80fac2a182cfba3f07f00/&can_dock=true&toolbarColor=rgba(223,223,223,1)&textColor=rgba(0,0,0,1)&experiments=true

**Comment by jrieken** [comment]
This is so weird... Can you use F1 > Dev Tools > Elements and find the containing `split-view-view` and see what its height is? Also, please try to drag the separator between tree and editor and see if that restores things

<img width="1229" alt="Screenshot 2019-04-29 at 11 56 06" src="https://user-images.githubusercontent.com/1794099/56889091-d512ae80-6a75-11e9-91cd-f4cb3af76f0f.png">

**Comment by Xarkam** [comment]
During my screenshots, I put the devtools in windowed mode and then I docked the devtools.
This action emptied the preview window.
Before resize
![image](https://user-images.githubusercontent.com/331816/56889613-21122300-6a77-11e9-88af-27ccd78f4a52.png)
After resize
![image](https://user-images.githubusercontent.com/331816/56889714-71898080-6a77-11e9-817b-caeea0850b6b.png)

Resizing the separator.
When the window is empty, I have no separator.

I moved the separator before resizing the window and I can not reproduce the bug anymore.

I will reset stable vscode. To validate this behavior.
